Xamarin.Forms.Core/ISliderController.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Xamarin.Forms
+{
+	public interface ISliderController
```

Comment:
require an EditorBrowsable.Never attribute

Reply:
@StephaneDelcroix Just wanted to confirm this is necessary? This is not used elsewhere on similar interfaces eg. `ISwipeGestureController`, `IScrollViewController` etc.

Reply:
@StephaneDelcroix Added.